src/base/bittorrent/session.cpp

```diff
@@ -155,6 +155,19 @@ namespace
 
         return expanded;
     }
+
+    QStringList findAllFiles(const QString &folderPath)
```

Comment:
IMO, the function would be neater with `directoryPath` or `dirPath` as the parameter name, because of the `QDir` inside of the function.